src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$calendar-day-size: 98px;
+$calendar-day-border-size: 1px;
+$calendar-horizontal-padding: 10px;
+$calendar-day-gap: 1px;
+
+$calendar-day-width: $calendar-day-size + $calendar-day-border-size * 2;

```

Comment:
set `$calendar-day-size: 100px;` and `$calendar-day-width: $calendar-day-size * 2;`